service/src/main/kotlin/no/nav/su/se/bakover/service/søknad/SøknadServiceImpl.kt

```diff
@@ -26,46 +28,81 @@ internal class SøknadServiceImpl(
         return søknadRepo.hentSøknad(søknadId)?.right() ?: KunneIkkeLukkeSøknad.FantIkkeSøknad.left()
     }
 
-    private fun lukkSøknad(
+    override fun lukkSøknad(
         søknadId: UUID,
         saksbehandler: Saksbehandler,
-        begrunnelse: String,
-        loggtema: String
+        lukketSøknadBody: Søknad.LukketSøknadBody
+    ): Either<KunneIkkeLukkeSøknad, Sak> {
+        return trekkSøknad(
```

Comment:
Jeg er heller for å starte spesifikt, for så og bli generisk, enn omvendt. Det var det jeg prøvde å vise i forrige PR.